### ISSUE TITLE
Publishing goes to home page for missing pages

### ISSUE THREAD

**Issue by SeriousBug** [dendronhq/dendron#2083]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [x] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

When opening the page for a note that's missing, the published site automatically navigates to the home page.

## To Reproduce

1. Click on this link: [https://wiki.dendron.so/notes/does-not-exist](https://wiki.dendron.so/notes/does-not-exist)
2. Site navigates to the home page.

## Expected behavior

It should display a 404 page.

## Desktop (please complete the following information)

- OS: Linux
- Browser: Firefox

**Comment by kevinslin** [comment]
yeah, this is an issue. i've added it to [[01 Publish Seed2tree|dendron://private/proj.2022.01-publish-seed2tree]]
